### PR TITLE
Update "How to become a Beatmap Nominator" section

### PR DESCRIPTION
**Pull request by Okorin**
+ ported thread
+ ported images (made them white as wiki is dark gray so having black was lol)
+ adjusted wording for the things that _I think_ are up to date atm

### CODE REVIEW

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 1]
```diff
@@ -24,11 +24,39 @@ Roles and Responsibilities
...
--------------
+Becoming a Beatmap Nominator
+----------------------------
 
-With the change to the team on 22. August 2014, joining the BNG is now a lot easier than it used to be.
-For more details, see the forum post [How to join the BNG](https://osu.ppy.sh/forum/t/471344) on how to join the Beatmap Nomination Group.
+With the change to the team on 22. August 2014, joining the BNG is now a lot easier than it used to be in the past.
+The [Quality Assurance Team](https://osu.ppy.sh/g/7) will post an announcement for Beatmap Nominator Sign-ups to the [Pending Beatmaps Subforum](https://osu.ppy.sh/forum/6) in irregular intervals.
+
+#### Basic Criteria
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nomination_Group/en.md]: Skipped level heading.
Use `### Basic Criteria`

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 2]
```diff
@@ -24,11 +24,39 @@ Roles and Responsibilities
...
+Modders that aim to apply to the Beatmap Nominators must fulfill the following criteria:
++ They have been modding beatmaps actively for at least 3 months prior to the announcement.
++ They have accumulated at least 50 [Kudosu](wiki/Glossary/#kudosu) in total.
++ They provide 3 to 5 of their mods which they think qualify them to be a part of the Beatmap Nominators.
+
+#### What Qualifies as a Mod Post
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nomination_Group/en.md]: ditto; Skipped level heading.
Use `### What Qualifies as a Mod Post`

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 3]
```diff
@@ -24,11 +24,39 @@ Roles and Responsibilities
...
++ Upon signing up you will be asked to provide the total amount of Kudosu as well as the amount of mod posts within the last 3 months, sorted by month upon signing up. So if the announcement is posted in April the mod posts for January, February and March will be relevant. The relevant months will also be listed on the submission form.
++ A mod post qualifies as one if it has received a Kudosu and has been created within the time span listed on the submission form.
++ The gamemode (osu!, osu!mania, osu!catch, osu!taiko) that you modded beatmaps in does not matter as your activity in the modding community as a whole is being measured and not the activity in a certain game mode.
++ The point in time at which the Kudosu has been awarded to you mod post does not matter, the only relevant information about a mod post is its post date and time in UTC+0 as well as the fact that it has been awarded Kudosu.
+
+#### How Basic Score is Calculated
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nomination_Group/en.md]: ditto; Skipped level heading.
Use `### How Basic Score is Calculated`

**Comment by Okorin** [comment]
kill all dittos
made everything be lvl 3 headings

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 4]
```diff
@@ -24,11 +24,39 @@ Roles and Responsibilities
...
 
-With the change to the team on 22. August 2014, joining the BNG is now a lot easier than it used to be.
-For more details, see the forum post [How to join the BNG](https://osu.ppy.sh/forum/t/471344) on how to join the Beatmap Nomination Group.
+With the change to the team on 22. August 2014, joining the BNG is now a lot easier than it used to be in the past.
+The [Quality Assurance Team](https://osu.ppy.sh/g/7) will post an announcement for Beatmap Nominator Sign-ups to the [Pending Beatmaps Subforum](https://osu.ppy.sh/forum/6) in irregular intervals.
+
+### Basic Criteria
+Modders that aim to apply to the Beatmap Nominators must fulfill the following criteria:
++ They have been modding beatmaps actively for at least 3 months prior to the announcement.
++ They have accumulated at least 50 [Kudosu](wiki/Glossary/#kudosu) in total.
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nomination_Group/en.md]: missing `/` (root)
use `[Kudosu](/wiki/Glossary/#kudosu)`

**Comment by Ephemeralis** [comment]
do we have standards re: heading capitalization? I used standard case for help center, wondering if we should do the same here (it's currently title case as of this comment)

**Comment by MegaApplePi** [comment]
@Ephemeralis the current standard listed in the ASG is to use title case for all headings
chrome_2017-07-23_22-52-46
chrome_2017-07-23_22-53-11

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 5]
```diff
@@ -24,11 +24,39 @@ Roles and Responsibilities
...
++ This depends on the total `Kudosu Score` and the total `Mod Score` an applicant achieved over the last months. A mathematical formula is used to determine these values.
++ `Kudosu Score` is determined according to the following formula:
+
+![Kudosu Score Formula](/wiki/shared/KDSCORE.png "Kudosu Score Formula")
+
++ `Mod Score` per month is determined according to the following formula:
+
+![Mod Score Formula](/wiki/shared/MODSCORE.png "Mod Score Formula")
+
++ In order to determine their `Basic Score`, an applicant needs to summarise their `Kudosu Score` and their `Mod Score` for month one, two and three.
```
Comment (Loctav) [inline wiki/People/Beatmap_Nomination_Group/en.md]: Basic Score -> Required Minimum Score

**Comment by Okorin** [comment]
replaced Basic Score with Required Minimum score

since that's longer i worked around and avoided using it in multiple consecutive sentences

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 6]
```diff
@@ -24,11 +24,39 @@ Roles and Responsibilities
...
 BNs are the people to go to for any modding or advice requests about anything mapping or modding related.
 
-Becoming a BN
--------------
+Becoming a Beatmap Nominator
+----------------------------
 
-With the change to the team on 22. August 2014, joining the BNG is now a lot easier than it used to be.
-For more details, see the forum post [How to join the BNG](https://osu.ppy.sh/forum/t/471344) on how to join the Beatmap Nomination Group.
+With the change to the team on 22. August 2014, joining the BNG is now a lot easier than it used to be in the past.
```
Comment (TPGPL) [inline wiki/People/Beatmap_Nomination_Group/en.md]: BNG into Beatmap Nomination Group

**Comment by Okorin** [comment]
avoided short form

**Pull request merged**